src/Microsoft.AspNetCore.Mvc.Core/DisableRequestSizeLimitResourceFilter.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// A filter that sets the request body size limit to null.
+    /// </summary>
+    public class DisableRequestSizeLimitResourceFilter : IResourceFilter, IRequestSizePolicy
+    {
+        private readonly ILogger _logger;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="DisableRequestSizeLimitResourceFilter"/>.
+        /// </summary>
+        public DisableRequestSizeLimitResourceFilter(ILoggerFactory loggerFactory)
+        {
+            if (loggerFactory != null)
+            {
+                _logger = loggerFactory.CreateLogger<DisableRequestSizeLimitResourceFilter>();
+            }
+        }
+
+        /// <inheritdoc />
+        public int Order { get; set; }
+
+        /// <inheritdoc />
+        public void OnResourceExecuted(ResourceExecutedContext context)
+        {
+
+        }
+
+        /// <summary>
+        /// As an <see cref="IResourceFilter"/>, this filter sets the <see cref="IHttpMaxRequestBodySizeFeature.MaxRequestBodySize"/>
+        /// to null.
+        /// </summary>
+        /// <param name="context">The <see cref="ResourceExecutingContext"/>.</param>
+        /// <remarks>If <see cref="IHttpMaxRequestBodySizeFeature"/> is not enabled or is read-only, the attribute is not applied.</remarks> 
```

Comment:
This isn't an attribute. Therefore, `the attribute` -> `this filter`. Same thing a few more times below, especially in `RequestSizeLimitResourceFilter`.